tools/toolchains/gcc.py

```diff
@@ -273,13 +273,13 @@ def __init__(self, target, options=None, notify=None, macros=None, silent=False,
         GCC.__init__(self, target, options, notify, macros, silent, TOOLCHAIN_PATHS['GCC_ARM'], extra_verbose=extra_verbose)
 
         # Use latest gcc nanolib
-        if "big-build" in self.options:
+        if "std-lib" in self.options:
```

Comment:
As this breaks a user space (if they use big-build) I would do it non-breaking way for now, or at least to print a warning "use this new option, this is not supported anymore".  I would go with non-breaking way